src/v2/api/group/GroupService.ts

```diff
@@ -69,4 +54,26 @@ export class GroupService {
 
     return results;
   }
+
+  async getCaptain(groupId: string) {
+    const students = await this.groupRepository.getStudents(groupId);
+    for (const student of students) {
+      const roles = await this.studentRepository.getRoles(student.userId);
+      for (const role of roles){
+        if (String(role) == "captain"){
+          return student;
+        }
+      }
+    }
+
+  }
+
+  async deleteGroup(groupId: string) {
+    await this.groupRepository.delete(groupId);
+  }
+
+  async getStudents(groupId: string) {
+    await this.groupRepository.getStudents(groupId);
```

Comment:
You haven't return anything here :<. Also you need to clear the returning objects from fields, that will not be shown and add fields from user entity (email, for example)

Reply:
you can see an example in getDisciplines method